webos-appinfojson/subDirCase/app1/testResources.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * testResources.js - test file to verify generated resources.
+ *
+ * Copyright © 2023 JEDLSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var fs = require("fs");
+var path = require("path");
+var defaultRSPath = path.join(process.cwd(), "resources");
+
+function logResults(testname, expected, actual) {
+    if (expected === actual) {
+        console.log(testname + " is passed.");
```

Comment:
Very minor nitpick: it should be "has" instead of "is"

Reply:
oh,I  fixed the same cases in other files too. Thank you!